bin/build-fonts.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import SYSTEM_FONTS from './systemFonts';
+
+const { readFileSync, writeFileSync, existsSync } = require('fs');
+const process = require('process');
+
+const PLUGIN_DIR = process.cwd();
+const FONTS_FILE = PLUGIN_DIR + '/includes/data/fonts.json';
+
+if (!existsSync(FONTS_FILE)) {
+  process.exit(1);
+}
+
+const rawData = readFileSync(FONTS_FILE);
+const googleFonts = JSON.parse(rawData);
+
+const fonts = [...SYSTEM_FONTS];
+
+const FONT_VARIANT_PATTERN = /(?<weight>\d+)?(?<style>\D+)?/;
+
+/**
+ * Returns a valid CSS font fallback declaration for a Google font.
+ *
+ * @param {string} category Font category.
+ * @return {string} Font fallback.
+ */
+function getFontFallback(category) {
+  switch (category) {
+    case 'sans-serif':
+      return 'sans-serif';
+    case 'handwriting':
+    case 'display':
+      return 'cursive';
+    case 'monospace':
+      return 'monospace';
+    default:
```

Comment:
Nit: Could combine `monospace` and `sans-serif` by
```
case `sans-serif`:
case `monospace`:
    return category;
```

Reply:
Why not 👍 

Copied this 1:1 from the current PHP implementation, but sounds like a nice little improvement.